samples/hello-oboe/src/main/cpp/PlayAudioEngine.cpp

```diff
@@ -269,23 +215,16 @@ PlayAudioEngine::calculateCurrentOutputLatencyMillis(oboe::AudioStream *stream,
  * @see oboe::StreamCallback
  */
 void PlayAudioEngine::onErrorAfterClose(oboe::AudioStream *oboeStream, oboe::Result error) {
-    if (error == oboe::Result::ErrorDisconnected) restartStream();
+    if (error == oboe::Result::ErrorDisconnected) {
+        oboe::AudioStreamBuilder builder_ = oboe::AudioStreamBuilder(*oboeStream);
+        restartStream(&builder_);
```

Comment:
Do we leak the old AudioStream pointer?
If you delete it then you may hit this issue!
https://github.com/google/oboe/issues/519

Reply:
Restarting will use the same pointer, `mPlayStream`, but create a new object. The old object is closed but not deleted, however, I think this issue should be addressed by using a unique pointer, which involves issue #529